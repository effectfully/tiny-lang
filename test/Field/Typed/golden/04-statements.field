let x : field = 1;
assert T;
for x = 1 to 1 do end;
