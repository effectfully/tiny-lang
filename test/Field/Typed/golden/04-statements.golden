Program [] Statements [ELet (UniVar {_uniVarUni = Field, _uniVarVar = x_1}) (EConst 1),EAssert (EConst (UniConst Bool True)),ELet (UniVar {_uniVarUni = Field, _uniVarVar = x_2}) (EConst 1)]